addons/mail/static/src/messaging/component/chatter/chatter.scss

```diff
@@ -9,32 +9,27 @@
     overflow: auto;
 }
 
-.o_FormRenderer_chatterContainer {
-    display: flex;
-    flex: 1 1 auto;
-    margin-left: auto;
-    margin-right: auto;
-    width: 100%;
+.o_Chatter_composer {
+    border-bottom: 1px solid;
 
-    @media (min-width: 576px) {
-        max-width: 540px;
-    }
-
-    @media (min-width: 768px) {
-        max-width: 720px;
-    }
-
-    @media (min-width: 992px) {
-        max-width: 960px;
+    &.o-bordered {
+        border-left: 1px solid;
+        border-right: 1px solid;
     }
+}
 
-    @media (min-width: 1200px) {
-        max-width: 1140px;
-    }
+/* Specificity needed to override generic style made in form_view.scss */
+.o_form_view .o_form_sheet_bg .o_FormRenderer_chatterContainer {
+    margin-left: auto;
+    margin-right: auto;
+}
 
-    @media (min-width: 1534px) {
-        width: 530px;
-    }
+.o_FormRenderer_chatterContainer {
+    display: flex;
+    flex: 1 1 auto;
+    margin: 0 auto;
+    max-width: 1140px;
```

Comment:
This should likely be related to `$sheet-max-width`

Reply:
This variable is scoped I think because it's not available in `chatter.scss`

Reply:
I would be in favor of making it global instead of scoped in its original file. Maybe then renamed to something similar to `$form-view-sheet-max-width` to keep it's "namespace".

Reply:
And with `!default` to allow customization while we are at it.